### PR TITLE
Perl cpan img

### PR DESCRIPTION
**Pull request by tsachleben**

### CODE REVIEW

**Comment by tsachleben** [comment]
Should be reviewed after #3 (dependent on those changes)

File: docker-images/get-info/src/perl5/Reporting/DB/Dump.pm [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+    return; 
+    my $smt = $self->prepare("select project_id from poc2moc_project");
+    my $ins = $self->prepare("insert into poc2moc_project (poc_id, moc_project_id, poc_poc_id, role_id, username) values (?,?,?,?,?)");
+    foreach my $user_id (keys %{$obj->{poc2moc_project}})
+    {
+        foreach my $project_id (keys %{$obj->{poc2moc_project}->{$user_id}})
+        {
+            # TODO: $ins select a poc_poc_id 
+            # Note: Is this field even necessary? ~TS
```
Comment (rob-baron) [inline docker-images/get-info/src/perl5/Reporting/DB/Dump.pm:100]: It seems that there is a bit of confusion as to the roles.

  there is an institutional role (institution2poc.role_id)
      Business Administrator
      Primary Investigator
      Software Engineer
      ....

There is a role on the service project (like an OpenShift project or an OpenStack project)
This is encoded in poc2project.role_id:
    admin
    member
    view

We also need to combine various projects under ESI, OpenShift and OpenStack under a single reportable project.  This is in part a place holder until the business logic can be complete worked out, but here is what I foresee:
    1) admin - can create projects on any of the services and invite project level admins, members and viewers.
    2) business administrators - can get reports on the subset of projects that they are responsible for.  There are business adminstrors at the institution level for each research group, however, we should not send all finiancial reports to all business administrators on every project.  We also don't want to for business adminstrators to consolidate all of the financials from individual reports we could send them if we included the role at the project level.
    3) member - can view all of the projects and the status of the projects, but cannot create new projects and cannot add themselves to any particular project.

File: docker-images/get-info/src/perl5/Reporting/DB/Role.pm [hunk 2]
```diff
@@ -0,0 +1,71 @@
+
+package Reporting::DB::Role;
```
Comment (rob-baron) [inline docker-images/get-info/src/perl5/Reporting/DB/Role.pm:2]: My expectation is that role will be a static table that we will be reading from quite often, but updating infrequently.

**Review by rob-baron — APPROVED** [review]
Ok, Although I don't particularly care for the ORM lite approach you seem to be taking, for an initial cut it is fine.  I am also not really a fan for doc strings, but as long as they produce a coherent document, that is all that matters.

There are some sections that we will need to optimize a bit, but we can deal with that once we get data.

**Pull request merged**